src/index.js

```diff
@@ -5,12 +5,26 @@ export default function (Alpine) {
 
       const hasDynamicTemplate = this.hasAttribute(':template')
       const hasDynamicUrl = this.hasAttribute(':url')
+      const hasDynamicStyles = this.hasAttribute(':styles')
```

Comment:
I don't think we need to support `:styles` so this can be removed.

Reply:
I guess we could enable it by default, but I'd consider this a breaking change if users were already using the workaround. Also maybe it's be worth to investigate if we can emulate the hierarchy rules if the aim is to make a truly *transparent* support 🤔 ?

Reply:
Attributes with `:` at the start are for values that will be passed as a variable. It's not something I see people doing and has been removed in the commit I did the other day. 

As for the hierarchy rules I think leave it for now and see if people request it.